paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -109,11 +109,14 @@ internal class DefaultFlowController @Inject internal constructor(
     override var shippingDetails: AddressDetails?
         get() = viewModel.state?.config?.shippingDetails
         set(value) {
-            viewModel.state = viewModel.state?.copy(
-                config = viewModel.state?.config?.copy(
-                    shippingDetails = value
+            val state = viewModel.state
+            if (state != null) {
```

Comment:
Why is this different?

Reply:
We're updating the config only, and the config looks nullable in the new version (because state is nullable). But this didn't change any behavior.